examples/4337-passkeys/src/logic/userOp.ts

```diff
@@ -77,6 +59,9 @@ const DUMMY_SIGNATURE = ethers.solidityPacked(
   ],
 )
 
+const DUMMY_SIGNATURE =
```

Comment:
Nit: You should build this with the dummy signature from the `passkey` package:

```ts
function dummySignatureUserOp(signer: string) {
  return ethers.solidityPacked(
    ['uint48', 'uint48', 'bytes'],
    [0, 0, buildSignatureBytes([
      {
        signer,
        data: DUMMY_SIGNATURE, // imported from passkey project
        dynamic: true,
      },
    ])],
  )
}
```

This is also important, as otherwise you may potentially be using an invalid estimate (for example, if we change the verifiers to only use precompile, the estimate will be way off, but we won't catch it with the current code).

Reply:
And in fact, the signature estimation greatly depends on the `verifiers` configuration that is used.